cubedash/_pages.py

```diff
@@ -195,7 +195,7 @@ def search_page(
     # TODO: Add sort option to index API
     datasets = sorted(
         _model.STORE.index.datasets.search(**query, limit=_HARD_SEARCH_LIMIT + 1),
-        key=lambda d: d.center_time,
+        key=lambda d: d.center_time.replace(tzinfo=pytz.utc),
```

Comment:
This will cause the value to be wrong for non-UTC timezones (which, admittedly, we rarely see)

Ideally it should call [default_utc()](https://github.com/opendatacube/datacube-explorer/blob/7d67ad90e5296b0bf8accace706763cee1293739/cubedash/_utils.py#L391-L394) to only add the timzone if needed